assets/railcraft/lang/ru_RU.lang

```diff
@@ -1073,3 +1156,113 @@ tile.railcraft.wall.stone.brick.name=Стена из каменного кирп
+track_kit.railcraft.railcraft_embarking.tips=Сажает существ в вагонетки\n§o§9- Ударьте монтировкой, чтобы изменить дальность -\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_gated.name=Путь с воротами
+track_kit.railcraft.railcraft_gated.tips=Путь со встроенными воротами\n§o§c- Необходим источник Redstone-сигнала чтобы открыть -
+
+track_kit.railcraft.railcraft_gated_one_way.name=Путь с воротами (односторонний)
+track_kit.railcraft.railcraft_gated_one_way.tips=Путь со встроенными воротами пропускает только в одну сторону\n§o§c- Необходим источник Redstone-сигнала чтобы открыть -
+
+track_kit.railcraft.railcraft_launcher.name=Запускатель
+track_kit.railcraft.railcraft_launcher.tips=Заставляет вагонетки летать!\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_throttle.name=Дроссель
+track_kit.railcraft.railcraft_throttle.tips=Настраивает скорость\n проходящих локомотивов\n§o§9- Ударьте монтировкой, чтобы изменить режим -\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_locking.name=Запиратель
+track_kit.railcraft.railcraft_locking.tips=Останавливает и удерживает вагонетки\n§o§9- Ударьте монтировкой, чтобы изменить режим -\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_locomotive.name=Вылючатель локомотивов
+track_kit.railcraft.railcraft_locomotive.tips=Запускает останавливает локомотивы\n§o§9- Ударьте монтировкой, чтобы изменить режим -\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_missing.name=Неподходящий
+track_kit.railcraft.railcraft_missing.tips=Пустое описание
+
+track_kit.railcraft.railcraft_one_way.name=Односторонний ограничитель
+track_kit.railcraft.railcraft_one_way.tips=Вагонетки могут проходить\n только по направлению стрелки\n§o§9- Ударьте монтировкой, чтобы изменить направление -
+
+track_kit.railcraft.railcraft_priming.name=Воспламенитель
+track_kit.railcraft.railcraft_priming.tips=Поджигает вагонетки со взрывчаткой\n Изменяемый запал\n§o§9- Ударьте монтировкой, чтобы изменить запал -\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_routing.name=Маршрутизатор
+track_kit.railcraft.railcraft_routing.tips=Устанавливает проходящему локомотиву билет\n§o§9- Ударьте монтировкой, чтобы изменить билет -
+
+track_kit.railcraft.railcraft_transition.name=Переход
+track_kit.railcraft.railcraft_transition.tips=Переводит на/с Высокой скорости\n§o§c- Необходим источник Redstone-сигнала -
+
+track_kit.railcraft.railcraft_whistle.name=Активатор свистка
+track_kit.railcraft.railcraft_whistle.tips=Проходящие локомотивы\n подуют в свисток\n§o§c- Необходим источник Redstone-сигнала -
+
+track_type.railcraft.railcraft_iron.name=Железный
+track_type.railcraft.railcraft_strap_iron.name=Strap Iron
+track_type.railcraft.railcraft_abandoned.name=Заброшенный
+track_type.railcraft.railcraft_reinforced.name=Усилинный
+track_type.railcraft.railcraft_high_speed.name=H. S.
```

Comment:
Ok, can i use full size words? Or better make like in en_us with 1st letters of words?